scripts/updateReleaseNotes/markdown.js

```diff
@@ -0,0 +1,52 @@
+// @ts-check
+/// <reference path="./index.d.ts" />
+
+const { repoDetails } = require('./init');
+const { getPullRequest } = require('./pullRequests');
+
+const EOL = '\n';
+
+/**
+ * Build up the markdown from the entry description.
+ * This includes fetching info about relevant pull requests, which can be slow/expensive.
+ * @param {IChangelogEntry} entry
+ */
+async function getMarkdownForEntry(entry) {
+  const comments =
+    (await getChangeComments('Breaking changes', entry.comments.major)) +
+    (await getChangeComments('Minor changes', entry.comments.minor)) +
+    (await getChangeComments('Patches', entry.comments.patch));
+
+  return (comments || '*Changes not tracked*') + EOL + EOL;
+}
+
+/**
+ * Gets the release notes markdown corresponding to the comment array.
+ * @param {string} title - Section title (probably change type, major/minor/patch)
+ * @param {IChangelogComment[]} comments - Changelog comments for a version
+ */
+async function getChangeComments(title, comments) {
+  if (comments) {
+    const lines = ['## ' + title, ''];
+    for (const comment of comments) {
+      let line = `- ${comment.comment} (`;
+      if (comment.commit) {
+        // Prefer linking to the PR if we can find it (this is generally more useful than the commit)
```

Comment:
New: if possible, link to only the PR (not the commit) since the PR is a lot more useful.

Old:
![image](https://user-images.githubusercontent.com/5864305/63473301-d5222a00-c429-11e9-981f-e2179b7a9e6d.png)

New:

* Add lint rule banning describe/it.only (PR [#10194](https://github.com/OfficeDev/office-ui-fabric-react/pull/10194) by [@ecraig12345](https://github.com/ecraig12345))
* sample unknown PR commit ([commit](https://github.com))